cmd/geth-hdwallet/main.go

```diff
@@ -16,6 +16,15 @@ func main() {
 	flag.StringVar(&hdpath, "path", "", "HD path")
 	flag.Parse()
 
+	// Check for empty values
+	if mnemonic == "" {
```

Comment:
This line 20-22 is already handled by: https://github.com/stephenlacy/go-ethereum-hdwallet/blob/master/hdwallet.go#L65
If you update it to have just the hdpath check I'll merge

Reply:
Actually, the current cli is using `MustParseDerivationPath` which will panic when the path is invaid.
Can you update it to use `ParseDerivationPath` and handle the error the same way as here? https://github.com/stephenlacy/go-ethereum-hdwallet/blob/master/cmd/geth-hdwallet/main.go#L20
Thanks